store/storetest/bot_store.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func makeBotWithUser(ss store.Store, bot *model.Bot) (*model.Bot, *model.User) {
-	user := store.Must(ss.User().Save(model.UserFromBot(bot))).(*model.User)
+	user, _ := ss.User().Save(model.UserFromBot(bot))
```

Comment:
The correct conversion of it would be including a `require.Nil(t, err)` there too. Something like:
```go
user, err := ss.User().Save(model.UserFromBot(bot))
require.Nil(t, err)
```